packages/eslint-config-sentry-app/strict.js

```diff
@@ -40,6 +40,8 @@ module.exports = {
 
     'sentry/no-styled-shortcut': ['error'],
 
+    'sentry/no-react-hooks': ['error'],
```

Comment:
Let's move this to the default ruleset since we don't need to change any code for it.

Reply:
👍